### DISCUSSION TITLE
How to push/pull with a self assigned certificate?

### DISCUSSION THREAD

**Discussion by Obstbeeren** [extrawurst/gitui#2259] [Q&A]
![image](https://github.com/extrawurst/gitui/assets/81931294/40a57a30-f1c0-44a4-874f-b90bb70b4cc8)

I do have `http.sslverify` set to false.

**Comment by ryanjspears** [comment]
I may be having a similar issue when I go to push. It is asking me for my username and password.  Is this for my git signing?
